setup.py

```diff
@@ -53,10 +53,22 @@
     # Often, this is the same as your README, so you can just read it in from
     # that file directly (as we have already done above)
     #
+    # Optional if long_description is written in RST but required for
+    # plain-text or Markdown; if unspecified, "applications should attempt to
+    # render [the long_description] as text/x-rst; charset=UTF-8 and fall back
+    # to text/plain if it is not valid rst" (see link below)
+    #
```

Comment:
I think this is in the wrong place, it should be below `long_description_content_type=...`.

Reply:
Yeah, noticed just after pushing. Should be updated now.